src/pages/manageMembers.tsx

```diff
@@ -47,16 +47,13 @@ export const ManageMembers: React.FC = () => {
 
   // dao data
   const {data: daoDetails, isLoading} = useDaoDetailsQuery();
+  const pluginAddress = daoDetails?.plugins?.[0]?.instanceAddress as string;
+  const pluginType = daoDetails?.plugins?.[0]?.id as PluginTypes;
+
   // plugin data
-  const {data: pluginSettings} = usePluginSettings(
-    daoDetails?.plugins[0].instanceAddress as string,
-    daoDetails?.plugins[0].id as PluginTypes
-  );
-  const {data: daoMembers} = useDaoMembers(
-    daoDetails?.plugins?.[0]?.instanceAddress || '',
-    (daoDetails?.plugins?.[0]?.id as PluginTypes) || undefined
-  );
-  const multisigDAOSettings = pluginSettings as MultisigVotingSettings;
+  const {data: daoMembers} = useDaoMembers(pluginAddress, pluginType);
+  const {data: pluginSettings} = useVotingSettings({pluginAddress, pluginType});
+  const multisigVotingSettings = pluginSettings as MultisigVotingSettings;
```

Comment:
Same here, I would specify that the object might also be `undefined` at this point
```suggestion
  const multisigVotingSettings = pluginSettings as MultisigVotingSettings | undefined;
```